aws_ec2.yml

```diff
@@ -0,0 +1,5 @@
+plugin: aws_ec2
+regions:
+  - us-west-1
+filters:
+  "tag:Name": "loki"
```

Comment:
Also add tags for promtails so that we can install promtail agent on all VMs where every required